images/router/haproxy-base/Dockerfile

```diff
@@ -6,10 +6,16 @@
 #
 FROM openshift/origin-base
 
+#
+# Note: /var is changed to 777 to allow access when running this container as a non-root uid
+#       this is temporary and should be removed when the container is switch to an empty-dir
+#       with gid support.
+#
 RUN yum -y install haproxy && \
     mkdir -p /var/lib/containers/router/{certs,cacerts} && \
     mkdir -p /var/lib/haproxy/{conf,run,bin,log} && \
     touch /var/lib/haproxy/conf/{{os_http_be,os_edge_http_be,os_tcp_be,os_sni_passthrough,os_reencrypt}.map,haproxy.config} && \
+    chmod -R 777 /var && \
```

Comment:
No, this part will stay until we mod the router to use emptydir and change the router plugin to start looking in a different path